test-apps/database-test-app/src/main/java/com/google/firebase/testapps/database/TestActivity.java

```diff
@@ -81,15 +82,11 @@ public void onCancelled(DatabaseError error) {
     auth.signOut();
     auth.signInWithEmailAndPassword("test@mailinator.com", "password")
         .addOnSuccessListener(
-            new OnSuccessListener<AuthResult>() {
-              @Override
-              public void onSuccess(AuthResult authResult) {
+            authResult ->
```

Comment:
nit: strange indentation here, `authResult` is indented left of `public`...

Reply:
Okay. I was thinking maybe we can use lambda features since test-app now use Java 8.

Reply:
I'm not sure if we can use lambda (I don't see why not) I was just saying it looks like you need to move authResult in a few columns.  Not a big deal!

Reply:
Hmm.. I was using google-java-format as indent format for this file. And whenever I reformat, it goes like this. I don't know. Should I disable it?

Reply:
Nope!  `google-java-format` is smarter than I am